Documentation/compatibility/WCF-Improved-WCF-chain-trust-certificate-validation-for-Net.Tcp-certificate-authentication.md

```diff
@@ -0,0 +1,33 @@
+## Improved WCF chain trust certificate validation for Net.Tcp certificate authentication
+
+### Scope
+Minor
+
+### Version Introduced
+4.7.2
+
+### Source Analyzer Status
+NotPlanned
+
+### Change Description
+.NET Framework 4.7.2 made improvement on chain trust certificate validation when using certificate authentication with transport security with WCF. With this improvement, client certificates that are used to authenticate to a server must be configured for client authentication.  Similarly server certificates that are for the authenticating a server must be configured for server authentication. With this change if the root certificate is disabled, the certificate chain validation fails. The same change was also made to 3.5 and above released .NET Framework versions via Windows security roll-up. You can find more information [here](https://support.microsoft.com/en-us/help/4055269/security-only-update-for-net-framework-3-5-1-4-5-2-4-6-4-6-1-4-6-2-4-7).
```

Comment:
better: "made improvement on" --> "improves"
With this change --> comma after "change"
3.5 and above released .NET Framework versions --> ,NET Framework 3.5 and later versions